### PR TITLE
added drawer component and responsiveness to most pages

### PR DESCRIPTION
**Pull request by surbhi84**
besides cards, TU, navigation, grids and drawer, most pages are responsive .

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -34,3 +34,9 @@
   align-items: center;
   justify-content: center;
 }
+
+@media screen and (max-width: 480) {
```
Comment (rohanmathur91) [inline components/grid/grid.css]: unit is not mentioned

File: home-page/home.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+else column.classList.add("display-none");
+
+menuBtn.addEventListener("click", viewColumn);
+window.onresize = () => {
+  console.log(menuBtn.style);
+  if (window.screen.width >= 768) column.classList.remove("display-none");
+  else column.classList.add("display-none");
+};
+
+window.onload = window.onresize;
```
Comment (rohanmathur91) [inline home-page/home.js]: Suggestion: you can put comments here for what is happening

**Review by rohanmathur91 — APPROVED** [review]
Added few comments, rest looks good.

**Review by Creator-singh — APPROVED** [review]
looks good to go

**Pull request merged**